### PR TITLE
Add context parameter type

### PR DESCRIPTION
**Pull request by bryanjbelanger**
You can now add parameter elements to the context file.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -298,7 +298,7 @@ Style/EmptyLiteral:
 Metrics/LineLength:
   Enabled: False
 
-Style/MethodCallParentheses:
+Style/MethodCallWithoutArgsParentheses:
```
Comment (tphoney) [inline .rubocop.yml]: This should not be part of this pull request.

**Comment by tphoney** [comment]
Can you please have a look at the failing acceptance tests. Many thanks for the PR

**Comment by tphoney** [comment]
Hi @bryanjbelanger , thanks for the work you have put into this pull request. However due to inactivity and failing tests  this pull request is being closed. If you want to re-submit this PR with the necessary changes, we will gladly look at the new pull request.

**Comment by joshbeard** [comment]
I resurrected this pull request with very minor updates with passing tests and opened PR #317 with those changes (I don't think it's possible for me to modify _this_ pull request).
